src/Pages/WhyVacateEmployment.jsx

```diff
@@ -1,10 +1,163 @@
+  },
+  {
+    summary: "Online marketing professionals",
+    details:
+      "The surge in the number of businesses using online marketing in the last decade is a huge advantage for job seekers. Someone with a felony who also has a reasonable level of knowledge and experience would have no problem with background checks. In most cases, if the crime of their felony is not related to online frauds and manipulations, hiring won’t be a complicated process.",
+  },
+  {
+    summary: "Oil and gas professionals",
+    details:
+      "The oil and gas industry offers endless opportunities for job seekers at any level.",
+  },
+  {
+    summary: "App and software development companies",
+    details:
+      "Mobile applications and software developers are some of the highly sought-after professionals in the United States. Every business that takes online marketing seriously has mobile apps. Technology companies that offer mobile app development and software services are always on the lookout for the best talents in the industry. If you have what it takes, your felony will not always come in your way of getting jobs like this one.",
+  },
+  {
+    summary: "Computer repair technician jobs",
+    details:
+      "Computer repair technicians are estimated to grow by more than 8.2 percent over the next decades. In 2016 alone, the number of computer technicians was 69,030 in California, 43,800 in New York, 21,700 and 6,420 in North Carolina and South Carolina, respectively, as well as over 500,000 more all over the United States.",
+  },
+  {
+    summary: "Retail companies",
+    details:
+      "Retail companies will hire people with felonies, but are less likely to hire people with a records of theft or violence.",
+  },
+  {
+    summary: "Transport companies",
+    details:
+      "People with felonies can find job opportunities in the transport industry as drivers, carriers, and delivery guys.",
+  },
+  {
+    summary: "Food services and grocery stores",
+    details: "Missing",
+  },
+];
 
 const WhyVacateEmployment = () => {
+  const classes = WhyVacateEmploymentStyles();
   return (
-    <Container>
-        <Typography variant="h2">EMPLOYMENT</Typography>
-    </Container>
+    <>
+      <Container maxWidth="lg">
+        <Grid container alignItems="flex-start">
+          <Typography className={classes.title} variant="h2">
+            Employment barriers
+          </Typography>
+          <Typography className={classes.contentText} variant="body1">
+            Simply having a conviction does not prevent you from having a job,
+            however, it can be more difficult depending on the type of job and
+            the employer. It is important to not lose hope while facing
+            challenges such as:
+          </Typography>
+        </Grid>
+        <FactGroup fact1={unemployment} fact2={rejected} fact3={education} />
```

Comment:
It's from the TextWithImage file. I didn't want to alter something you might be using in your branch. 

```
return (
        <Grid item xs={12} sm={6} md={6}>
            <Box display="flex" justifyContent="center">
                <Box component="img" src={img} className={classes.roleImage} />
            </Box>
            <Typography
                className={classes.headingRoles}
                variant="h3"
                align="center"
                >
                {header}
            </Typography>
            { textItems }
        </Grid>
```

If we removed the grid item to outside the implementation or pass some props to allow other options then I can make it three columns not two and it will look right.

Reply:
Ah okay, I see. Let's add a prop to allow for either 2 col or 3 col? How does that sound?

Reply:
Sure. We can patch it once your branch is in.